org.jacoco.build/pom.xml

```diff
@@ -554,6 +555,7 @@
             <configuration>
               <target>
                 <fileset dir="${basedir}" includes="**/*.java,**/*.xml,**/*.bsh" excludes="target/**,.idea/**,nb-configuration.xml" id="missinglicense.fileset">
+                  <include name="**/*.kt"/>
```

Comment:
Same as above: New property for additional files to check?

Reply:
@marchof same as for above: here we define generic behavior for source files extensions,  highly unlikely, I would even say impossible, that same extension will have different behavior. `bsh` also not presented in all modules. IMO another way is far more confusing, more complex than really needed and error prone. Let's keep it simple.

Reply:
Ok!